app/models/tag_definition.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
-# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
-class TagMaster < ApplicationRecord
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/master/LICENSE
```

Comment:
This URL should be `https://github.com/houdiniproject/houdini/blob/main/LICENSE` as it is in the upstream change.